nautobot/dcim/tests/test_views.py

```diff
@@ -389,6 +445,30 @@ def test_list_rack_elevations(self):
         response = self.client.get(reverse("dcim:rack_elevation_list"))
         self.assertHttpStatus(response, 200)
 
+    @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
+    def test_powerports(self):
+        # Create Power Port for device
+        powerport1 = PowerPort.objects.create(device=self.devices[0], name="Power Port 11")
+        powerfeed1 = PowerFeed.objects.create(power_panel=self.powerpanels[0], name="Power Feed 11", phase="three-phase")
+        
+        # Create power outlet to the power port
+        poweroutlet1 = PowerOutlet.objects.create(device=self.devices[0], name="Power Outlet 11")
+        
+        # connect power port to power feed (3 phase)
+        cable1 = Cable(termination_a=powerfeed1, termination_b=powerport1, status=self.status_active)
```

Comment:
"Active" isn't a valid status for a cable. You should probably be using "Connected" instead.

Reply:
Do we know what the slug may be for this? `connected` was not one.